controllers/components/feastoperator/feastoperator_controller.go

```diff
@@ -42,6 +43,7 @@ func (s *componentHandler) NewComponentReconciler(ctx context.Context, mgr ctrl.
 		// Add FeastOperator-specific actions
 		WithAction(initialize).
 		WithAction(devFlags).
+		WithAction(releases.NewAction()).
```

Comment:
Probably, it should also implement WithReleases interface.

Thoughts: This action is common for all the components. I'm wondering would it be possible somehow to make a more common wrapper. It mus be called after manifests configured from the devFlags. Another question, if it should be done like that. I mean, may be release information should not be substituted with devFlags.

Reply:
two parts from me:
1. we can move the release releated into something as you suggested WithRelease() 
originally, i see it is close to updatestatus  but even more close to the way we set the release info from DSC/DSCI which is a one time action (this goes away after point (2) comes in) 
in short: we can refactor later. for now, it is functional as expected.
2. i had  thoughts on this order as well.  i convinced myself eventually, it needs to be done after devFlags. the reason is, if component want to test different value by having a mock value in the yaml file, so they can easily know the one running in the cluster is a mock/dev manifests not the original one from the operator build.
and with some ongoing support multi-arch work, that will be even helpful for the test purpose.

Reply:
> 1. we can move the release releated into something as you suggested WithRelease()

I mean, the action requires Feast to implement the interface https://github.com/opendatahub-io/opendatahub-operator/blob/main/pkg/controller/actions/status/releases/action_fetch_releases_status.go#L63

like https://github.com/opendatahub-io/opendatahub-operator/blob/main/apis/components/v1alpha1/codeflare_types.go#L76

Reply:
> 1. for now, it is functional as expected.

wondering how. Did I look wrong code?

Reply:
i missed this part, thought i added it in the push.........................

Reply:
@zdtsw you are also missing the updated generated files.